includes/reader-activation/class-reader-activation.php

```diff
@@ -240,6 +241,15 @@ public static function update_setting( $key, $value ) {
 		return \update_option( self::OPTIONS_PREFIX . $key, $value );
 	}
 
+	/**
+	 * Check if the required Woo plugins are active.
+	 *
+	 * @return boolean True if all required plugins are active, otherwise false.
+	 */
+	public static function is_woocommerce_active() {
```

Comment:
There's already the `Donations::is_woocommerce_suite_active` method – it should be used instead.

Reply:
on top of that, that method also checks for the "Name your price" plugin. We should add this to the messaging as well.

Reply:
Or maybe just abstract woo-checking plugin list, since it's [also used in the Reader Revenue wizard](https://github.com/Automattic/newspack-plugin/blob/39a247459b4151901799093e8bcd311d4f97abe7/assets/wizards/readerRevenue/views/platform/index.js#L55-L60).

Reply:
The checks don't necessarily overlap, though, because Reader Activation doesn't need Name Your Price or the Stripe Gateway extension. It only needs WC and WC Subscriptions if the Stripe platform is being used. So I think it's okay to keep these as separate methods and checks.